addons/hr_holidays/views/hr_views.xml

```diff
@@ -234,6 +234,50 @@
         </field>
     </record>
 
+    <record id="hr_kanban_view_employees_inherit_hr_holidays" model="ir.ui.view">
+        <field name="name">hr.kanban.view.employees.inherit.hr.holidays</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.hr_kanban_view_employees"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//kanban" position="attributes">
+                <attribute name="js_class">hr_holidays_kanban_view</attribute>
+             </xpath>
+        </field>
+    </record>
+
+    <record id="hr_employee_tree_inherit_hr_holidays" model="ir.ui.view">
+        <field name="name">hr.employee.tree.inherit.hr.holidays</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_tree"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//tree" position="attributes">
+                <attribute name="js_class">hr_holidays_list_view</attribute>
+             </xpath>
+        </field>
+    </record>
+
+    <record id="action_open_accrual_plan_employees" model="ir.actions.act_window">
+        <field name="name">Accrual Plan's Employees</field>
+        <field name="res_model">hr.employee</field>
+        <field name="view_mode">kanban,tree,form</field>
+    </record>
+
+    <record model="ir.actions.act_window.view" id="action_hr_kanban_view_employees_hr_holidays">
```

Comment:
Some nitpicking, but I think we should keep the order between model and id consistent :) 
```suggestion
    <record id="action_hr_kanban_view_employees_hr_holidays" model="ir.actions.act_window.view">
```